codegen/protocol-test-codegen-local/Tests/swift-codegen/WaitersTests/WaiterModels+Equatable.swift

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
The hard-coded Equatable conformance for hand-maintained protocol test for waiters.